src/systemctl/systemctl-list-units.c

```diff
@@ -932,3 +932,230 @@ int verb_list_automounts(int argc, char *argv[], void *userdata) {
+                                "org.freedesktop.systemd1.Path",
+                                "Paths",
+                                &error,
+                                &reply,
+                                "a(ss)");
+        if (r < 0)
+                return log_error_errno(r, "Failed to get paths: %s", bus_error_message(&error, r));
+
+        r = sd_bus_message_enter_container(reply, SD_BUS_TYPE_ARRAY, "(ss)");
+        if (r < 0)
+                return bus_log_parse_error(r);
+
+        while ((r = sd_bus_message_read(reply, "(ss)", &condition, &path)) > 0) {
+                r = strv_extend(&conditions, condition);
+                if (r < 0)
+                        return log_oom();
+
+                r = strv_extend(&paths, path);
+                if (r < 0)
+                        return log_oom();
+
+                n++;
+        }
+
+        r = sd_bus_message_exit_container(reply);
+        if (r < 0)
+                return bus_log_parse_error(r);
+
+        *ret_conditions = TAKE_PTR(conditions);
+        *ret_paths = TAKE_PTR(paths);
+
+        return n;
+}
+
+static int output_paths_list(struct path_infos *ps) {
+        _cleanup_(table_unrefp) Table *table = NULL;
+        int r;
+
+        assert(ps);
+        assert(ps->items || ps->count == 0);
+
+        table = table_new("path", "condition", "unit", "activates");
+        if (!table)
+                return log_oom();
+
+        table_set_header(table, arg_legend != 0);
+        if (arg_full)
+                table_set_width(table, 0);
+
+        table_set_ersatz_string(table, TABLE_ERSATZ_DASH);
+
+        for (struct path_info *p = ps->items; p < ps->items + ps->count; p++) {
+                _cleanup_free_ char *j = NULL;
+                const char *unit;
+
+                if (p->machine) {
+                        j = strjoin(p->machine, ":", p->id);
+                        if (!j)
+                                return log_oom();
+                        unit = j;
+                } else
+                        unit = p->id;
+
+                r = table_add_many(table,
+                                   TABLE_STRING, p->path,
+                                   TABLE_STRING, p->condition,
+                                   TABLE_STRING, unit);
+                if (r < 0)
+                        return table_log_add_error(r);
+
+                if (strv_isempty(p->triggered))
+                        r = table_add_cell(table, NULL, TABLE_EMPTY, NULL);
+                else if (strv_length(p->triggered) == 1)
+                        r = table_add_cell(table, NULL, TABLE_STRING, p->triggered[0]);
+                else
+                        /* This should never happen, currently our path units can only trigger a
+                         * single unit. But let's handle this anyway, who knows what the future
+                         * brings? */
+                        r = table_add_cell(table, NULL, TABLE_STRV, p->triggered);
```

Comment:
Why not to always use `TABLE_STRV` ??

Reply:
It matters for JSON output, see 50098d87fbe677bd43cb7329b2e32b039401f35c.